### PR TITLE
fix(pattern): CTA and button default - no icon

### PR DESCRIPTION
**Pull request by greeshmareddyg**
### Related Ticket(s)

https://github.com/carbon-design-system/ibm-dotcom-library/issues/2356

### Description

CTA section - default button without Icon
added default state to CTA, button group components.

### Changelog

**Changed**

- Added default button type

### CODE REVIEW

**Comment by jeffchew** [comment]
@greeshmareddyg can you fix the merge conflicts?

**Comment by jeffchew** [comment]
@greeshmareddyg switching CTA to type `default` renders this:

Screen Shot 2020-06-01 at 2 42 13 PM

**Comment by greeshmareddyg** [comment]
@jeffchew I will fix it Jeff.

File: packages/react/src/components/ButtonGroup/__stories__/ButtonGroup.stories.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ export const Default = () => {
       copy: text(`Button ${i + 1}`, `Button ${i + 1}`),
       renderIcon:
         iconMap[
-          select(`Icon ${i + 1}`, iconOptions, iconOptions['Arrow Right'])
+          select(`Icon ${i + 1}`, iconOptions, iconOptions[0])
```
Comment (asudoh) [inline packages/react/src/components/ButtonGroup/__stories__/ButtonGroup.stories.js:48]: Given `iconOptions` is not an array, wondering if this change meant to be below:

```suggestion
          select(`Icon ${i + 1}`, iconOptions, iconOptions['Default'])
```

Comment (greeshmareddyg) [reply packages/react/src/components/ButtonGroup/__stories__/ButtonGroup.stories.js]: yeah resolved the conflicts and corrected the above, Thank you @asudoh

**Comment by jeffchew** [comment]
@greeshmareddyg one minor thing, I know that we're missing `defaultProps` for a bunch of these components, can you add those? Let's have `default` be the default for `type`.

**Comment by greeshmareddyg** [comment]
> @greeshmareddyg one minor thing, I know that we're missing `defaultProps` for a bunch of these components, can you add those? Let's have `default` be the default for `type`.

sure, I will add that @jeffchew

**Pull request merged**